packages/lesswrong/components/tagging/subforums/SubforumSubforumTab.tsx

```diff
@@ -186,6 +199,20 @@ const SubforumSubforumTab = ({tag, isSubscribed, classes}: {
           />
         </div>
       )}
+      {tag.subforumIntroPost && !hideIntroPost && (
+        <div className={classes.feedPostWrapper}>
+          <RecentDiscussionThread
+            key={tag.subforumIntroPost._id}
+            post={{ ...tag.subforumIntroPost, recentComments: [] }}
+            comments={[]}
```

Comment:
Is this to make sure we don't display any comments for the intro post?

Reply:
Yes. I was planning to split out a separate PostCard component from RecentDiscussionThread for this, but from talking to JP it seems like we're planning on refactoring RecentDiscussionThread a lot in the near future so I thought it wasn't worth doing at this point